mantle/platform/qemu.go

```diff
@@ -1771,6 +1801,89 @@ func (builder *QemuBuilder) Exec() (*QemuInstance, error) {
 		return nil, err
 	}
 
+	// Process 9p/virtiofs mounts
+	if len(builder.hostMounts) > 0 && builder.UseVirtiofs {
+		if err := builder.ensureTempdir(); err != nil {
+			return nil, err
+		}
+
+		plog.Debug("creating viritofs helpers")
+
+		// Allocate a shared memory object and set up NUMA, which is needed for virtiofs
+		mempath, err := findMemoryBackendTarget(uint64(builder.Memory) * 1024)
+		if err != nil {
+			return nil, err
+		}
+		builder.Append("-object", fmt.Sprintf("memory-backend-file,id=mem,size=%dM,mem-path=%s,share=on", builder.Memory, mempath), "-numa", "node,memdev=mem")
```

Comment:
A few questions here:

- In we https://github.com/coreos/coreos-assembler/pull/3595#issuecomment-1707116355 dropped the NUMA bits. Does the same reasoning apply here which means we should drop the NUMA bits?
- There are now two places in this file that `Allocate a shared memory object` `which is needed for virtiofs`. I'm guessing we should drop one of them?